chroma-manager/chroma_core/management/commands/dev_setup.py

```diff
@@ -21,7 +21,7 @@ class Command(BaseCommand):
                     action='store_true',
                     dest='no_bundles',
                     default=False,
-                    help='Do not load any bundles, instead create dummies for use with simulator'),
+                    help='Do not load any bundles, instead create dummies'),
```

Comment:
not sure how any people actually use dev_setup any more may be used in some tests?

Reply:
Still used in service_tests. I wanted to get rid of this originally, but am considering it a transitional step for the moment.